python/lsst/ts/wep/task/cutOutDonutsBase.py

```
@@ -672,6 +675,10 @@ def cutOutStamps(self, exposure, donutCatalog, defocalType, cameraName):
             isEffective.append(eff)
             stampsEntropy.append(entro)
 
+            # Calculate fraction of bad pixels
+            bits = finalStamp.mask.getPlaneBitMask(("SAT", "BAD", "NO_DATA"))
```

We could set this dictionary of default mask plane bits as a config here in `cutOutDonuts`, eg.  `badPixelMaskDefinitions`, so that if someone wants to consider some other values in the future it's easy to do

---

Eg https://github.com/lsst/ip_isr/blob/main/python/lsst/ip/isr/isrFunctions.py#L182  and `CR` may be a choice one may use too  , so we  ould have that the default is `"SAT", "BAD", "NO_DATA"`

---

Good idea!